web/client-api/src/main/java/io/deephaven/web/client/api/WorkerConnection.java

```diff
@@ -948,19 +1043,53 @@ private TypedTicket createTypedTicket(JsVariableDefinition varDef) {
         return typedTicket;
     }
 
+
+    /**
+     * Retrieve a widget using a variable definition. Creates a new export ticket to reference the variable.
+     * Intentionally races the ticket export with the variable export to parallelize the requests.
+     * 
+     * @param varDef Variable definition to get the widget for
+     * @return Promise of the widget
+     */
     public Promise<JsWidget> getWidget(JsVariableDefinition varDef) {
+        return getWidget(varDef, true);
+    }
+
+    /**
+     * Retrieve a widget using a variable definition. Creates a new export ticket to reference the variable.
+     * Intentionally races the ticket export with the variable export to parallelize the requests.
+     * 
+     * @param varDef Variable definition to get the widget for
+     * @param refetch Whether to refetch the widget before resolving or not
+     * @return Promise of the widget
+     */
+    public Promise<JsWidget> getWidget(JsVariableDefinition varDef, boolean refetch) {
         return exportScopeTicket(varDef)
                 .race(ticket -> {
                     TypedTicket typedTicket = new TypedTicket();
                     typedTicket.setType(varDef.getType());
                     typedTicket.setTicket(ticket);
-                    return getWidget(typedTicket);
+                    return getWidget(typedTicket, refetch);
                 }).promise();
     }
 
-    public Promise<JsWidget> getWidget(TypedTicket typedTicket) {
+    /**
+     * Retrieve a widget using a ticket. Does not create a new ticket.
+     * 
+     * @param typedTicket Ticket to fetch
+     * @param refetch Whether to refetch the widget. If false, you must ensure you call {@JsWidget#refetch()} before
```

Comment:
```suggestion
     * @param refetch Whether to refetch the widget. If false, you must ensure you call {@link JsWidget#refetch()} before
```

(plus spotless, which will probably push a word to the next line)